Library/Formula/vobcopy.rb

```diff
@@ -14,6 +11,11 @@ def install
                              "--with-lfs",
                              "--with-dvdread-libs=#{HOMEBREW_PREFIX}"
     system "make"
-    system "make install"
+    system "make", "install"
+  end
+
+  test do
+    output = shell_output("#{bin}/vobcopy -h 2>&1", 1)
```

Comment:
I found something promising from [W6RZ](http://w6rz.net/): a DVD test-pattern disc image. But it's a ZIP of an ISO image. I don't know what to do to get at the ISO filesystem.